be/src/storage/chunk_helper.h

```diff
@@ -48,6 +48,9 @@ class ChunkHelper {
     static vectorized::Schema convert_schema_to_format_v2(const starrocks::TabletSchema& schema,
                                                           const std::vector<ColumnId>& cids);
 
+    // Get schema with format v2 type containing short key columns from TabletSchema.
+    static vectorized::Schema get_short_key_schema_with_format_v2(const starrocks::TabletSchema& schema);
```

Comment:
Currently, we are all format v2.

Reply:
It is to be consistent style with other methods, such as `convert_schema_to_format_v2`